olp-cpp-sdk-dataservice-read/src/VersionedLayerClientImpl.cpp

```diff
@@ -34,6 +34,8 @@
 #include "repositories/PartitionsRepository.h"
 #include "repositories/PrefetchTilesRepository.h"
 
+#include <algorithm>
```

Comment:
Please move up between own header and module headers.
See coding style guide -> https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes